great_expectations_cloud/agent/actions/generate_data_quality_check_expectations_action.py

```diff
@@ -524,6 +524,8 @@ def _create_expectation_for_asset(
         expectation_payload["autogenerated"] = True
         if created_via is not None:
             expectation_payload["created_via"] = created_via
+        # Automatic expectations are created with warning-level severity
+        expectation_payload["severity"] = "warning"
```

Comment:
Since `Expectations` have a `severity` parameter, rather than injecting a severity into the payload, it should be passed when the expectation is being created.

That change would have to wait until GX Core goes out.